.github/workflows/prober-prod.yml

```diff
@@ -21,6 +21,7 @@ jobs:
     uses: ./.github/workflows/reusable-prober.yml
     secrets:
       PAGERDUTY_INTEGRATION_KEY: ${{ secrets.PAGERDUTY_INTEGRATION_KEY }}
+      sigstore_github_token: ${{ secrets.SIGSTORE_BOT_PAT }}
```

Comment:
If there's an existing repository secret with sigstore-bot's PAT, I'll change the name here. If not, a repository maintainer will need to create it.